app/views/unattended/provisioning_templates/snippet/manage_packages.erb

```diff
@@ -0,0 +1,90 @@
+<%#
+kind: snippet
+name: manage_packages
+model: ProvisioningTemplate
+snippet: true
+-%>
+
+<%#
+  Snippet for :manage_packages macro
+  Usage in templates:
+    manage_packages('install', host_param('host_registration_install_packages'))
+    manage_packages('install', ['pkg1', 'pkg2]) or manage_packages('install', 'pkg1 pkg2')
+-%>
+<%
+  supported_families = ['Redhat', 'Debian', 'Suse']
+  render_error(N_('Unsupported or no operating system found for this host.')) unless @host.operatingsystem && supported_families.include?(@host.operatingsystem.family)
+
+  action = @action || 'install'
+  packages = @packages
+
+  if @host.operatingsystem.family == 'Redhat'
+    package_manager = 'yum'
```

Comment:
Note that on Fedora there is no yum and it's just dnf. Fedora is part of the Redhat OS family. On EL8 there is also dnf but yum is provided for convenience.

Reply:
But the yum is still there, right? I just set up fedora 31 image and can use yum without any problems.

Reply:
Fedora 31 is EOL. There is a `yum` package and it does look like it's installed by default on Fedora 33 which surprises me.

Reply:
Looks like it is just a symlink to dnf.

```shell
$ cat /etc/os-release | grep PRETTY_NAME                                                      
PRETTY_NAME="Fedora 33 (KDE Plasma)"

$ ls -la $(which yum)
lrwxrwxrwx. 1 root root 5 Dec  3 17:13 /usr/bin/yum -> dnf-3
```

While it may be there now, noone says it will still be there in future releases. I'd vote for not relying on it